src/osd/PGLog.h

```diff
@@ -979,7 +979,7 @@ struct PGLog : DoutPrefixProvider {
       ldpp_dout(dpp, 10) << __func__ << ": more recent entry found: "
 			 << *objiter->second << ", already merged" << dendl;
 
-      ceph_assert(objiter->second->version > last_divergent_update);
+      ceph_assert(objiter->second->version >= last_divergent_update);
```

Comment:
This looks pretty similar to https://tracker.ceph.com/issues/21174, I seem to recollect there was reason for this be > instead of >=, @dzafman @athanatos do you guys remember?